src/include/assert.h

```diff
@@ -66,8 +66,18 @@ struct BackTrace;
 extern void register_assert_context(CephContext *cct);
 #endif
 
+struct __ceph_assert_data {
```

Comment:
nit, quote from n4659, § 5.10
> Each identifier that contains a double underscore __ or begins with an underscore followed by an uppercase letter is reserved to the implementation for any use.


yes, i know we are using identifiers starting with two underscores already in this source file. but we can avoid doing so in new code.